.circleci/config.yml

```diff
@@ -291,14 +393,10 @@ jobs:
 
   # This job is manually triggered for now. see .circleci/README.md
   production-smoke-tests:
-    resource_class: medium+
-    docker:
-      - image: mcr.microsoft.com/playwright:v1.28.0-focal
+    executor: default-executor
     steps:
-      - base-install
-      - run:
-          name: Install playwright
```

Comment:
Thanks @dschom for all the comments and descriptions. I am just curious though, where are we doing an `install playwright` step for `production-smoke-tests` and `playwright-functional-tests` job?